spec/takeaway_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'takeaway'
+
+describe TakeAway do
+
+  subject(:takeaway) {described_class.new}
+
+  context 'Customer' do
+    it 'can see the menu' do
+      expect(takeaway).to respond_to :order
+    end
+    it 'can\'t order a dish not in the menu' do
+      expect{takeaway.add_meal :pomodoro, 2}.to raise_error 'not in the menu'
+    end
+    it 'can order a dish in the menu' do
+      expect{takeaway.add_meal :pizza,2}.to change{takeaway.order.length}.by 1
+    end
+    it 'can\'t delete a dish not in the ordered dishes' do
+      expect{takeaway.add_meal :pomodoro, 2}.to raise_error'not in the menu'
+    end
+    it 'can delete an item ordered' do
+      takeaway.add_meal :pizza, 2
+      expect{takeaway.delete :pizza,1 }.to change{takeaway.order[:pizza]}.by -1
```

Comment:
Lint/AmbiguousOperator: Ambiguous negative number operator. Parenthesize the method arguments if it's surely a negative number operator, or add a whitespace to the right of the `-` if it should be a subtraction. (https://github.com/bbatsov/ruby-style-guide#parens-as-args)